components/action-mgt/org.wso2.carbon.identity.action.mgt/src/main/java/org/wso2/carbon/identity/action/mgt/constant/ActionMgtConstants.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.action.mgt.constant;
+
+/**
+ * Constants for Action management service.
+ */
+public class ActionMgtConstants {
+
+    public static final String ACTIVE_STATUS = "ACTIVE";
+    public static final String INACTIVE_STATUS = "INACTIVE";
```

Comment:
Let's move this active, inactive to a proper status class or enum

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5771/commits/94e90e62016b434a0f37aa2b1f7ff0edc34bb64f